src/redisraft.h

```diff
@@ -778,6 +790,10 @@ void FreeImportKeys(ImportKeys *target);
 unsigned int keyHashSlot(const char *key, size_t keylen);
 unsigned int keyHashSlotRedisString(RedisModuleString *s);
 RRStatus parseHashSlots(char * slots, char * string);
+ClientState * GetClientState(RedisRaftCtx *rr, RedisModuleCtx * ctx);
```

Comment:
Regarding naming functions, I did many mistakes and merged bunch of things previously but trying to follow a style recently. So, for exported functions, function names start with uppercase letter and with a common prefix. So, maybe these functions should start with `Client..()`

Reply:
hmm.  So we want SubjectVerb not VerbSubject?

Reply:
Normally, in C world, people use a common prefix in exported function names (C version of namespaces). So, I guess it naturally becomes SubjectVerb because of the prefix. Good thing is that when you see that function name, you know where to find it. If a function starts with Shard... I know it's in cluster.c or with Conn.. it's in connection.c etc.